background/handle-fetch.js

```diff
@@ -1,59 +1,62 @@
-const extraInfoSpec = ["blocking", "requestHeaders"];
-if (Object.prototype.hasOwnProperty.call(chrome.webRequest.OnBeforeSendHeadersOptions, "EXTRA_HEADERS"))
-  extraInfoSpec.push("extraHeaders");
+if (chrome.declarativeNetRequest) {
```

Comment:
```suggestion
if (typeof chrome.declarativeNetRequest !== "undefined") {
```

Reply:
?

Reply:
Why do you want to check if this function returns string with value "undefined", not for undefined type (!== checks for type too)?

Reply:
> Why do you want to check if this function returns string with value "undefined", not for undefined type (!== checks for type too)?

`typeof` returns a string.